echopop/tests/test_kriging_biomass_apportionment.py

```diff
@@ -0,0 +1,878 @@
+                                                 categories = expected_output[ 'age_bin' ].unique( ) , 
+                                                 ordered=True)
+
+    #----------------------------------
+    ### Run tests: `compute_index_aged_weight_proportions`
+    #----------------------------------
+    ### Process the specimen data 
+    ### Check shape 
+    assert obj_props_wgt_len_age_sex.shape == expected_dimensions
+
+    ### Check datatypes
+    assert np.all( obj_props_wgt_len_age_sex.dtypes == expected_output.dtypes )
+    
+    ### Check data value equality
+    assert expected_output.equals( obj_props_wgt_len_age_sex )
+    
+def test_compte_aged_unaged_proportions( ):
+
+    ### Mock data for `specimen_data`
+    test_specimen_data = pd.DataFrame(
+        {
+            'stratum_num': np.repeat( [ 0 , 1 ] , 4 ) ,
+            'sex': np.tile( [ 'male' , 'female' ] , 4 ) ,
+            'haul_num': np.tile( [ 1 , 2 ] , 4 ) ,
+            'species_id': np.repeat( [ 19350 ] , 8 ) ,
+            'length': [ 12.0 , 12.0 , 19.0 , 19.0 , 12.0 , 12.0 , 19.0 , 19.0 ] ,
+            'weight': [ 2.0 , 3.0 , 8.0 , 7.0 , 1.0 , 4.0 , 9.0 , 6.0 ] ,
+            'age': [ 1 , 1 , 2 , 2 , 1 , 1 , 2 , 2 ]    
+        }
+    )
+
+    ### Mock data for `weight_strata`
+    test_weight_strata = pd.DataFrame(
+        {
+            'stratum_num': [ 0 , 1 ] ,
+            'weight_stratum_all': [ 50.0 , 100.0 ] ,
+        }
+    )
+
+    ### Evaluate for later comparison 
+    eval_aged_unaged_weight_proportions = compute_aged_unaged_proportions( test_specimen_data ,
+                                                                           test_weight_strata )
+    
+    ###--------------------------------
+    ### Expected outcomes
+    ###--------------------------------
+    ### `eval_aged_sex_proportions`
+    # ---- Expected dimensions
+    expected_dimensions = tuple( [ 2 , 7 ] )
+    # ---- Expected dataframe output
+    expected_output = pd.DataFrame( {
+        'stratum_num': np.array( [ 0.0 , 1.0 ] ).astype( np.int64 ) ,
+        'weight': [ 20.0 , 20.0 ] ,
+        'weight_stratum_all': [ 50.0 , 100.0 ] ,
+        'proportion_aged_weight_all': [ 0.40 , 0.20 ] ,
+        'proportion_aged_weight_adult': [ 0.30 , 0.15 ] ,
+        'proportion_unaged_weight_all': [ 0.60 , 0.80 ] ,
+        'proportion_unaged_weight_adult': [ 0.70 , 0.85 ]
+    } )
+
+    #----------------------------------
+    ### Run tests: `compute_index_aged_weight_proportions`
+    #----------------------------------
+    ### `eval_aged_sex_proportions`
+    # ---- Shape
+    assert eval_aged_unaged_weight_proportions.shape == expected_dimensions
+    # ---- Datatypes
+    assert np.all( eval_aged_unaged_weight_proportions.dtypes == expected_output.dtypes )
+    # ---- Dataframe equality
+    assert np.allclose( eval_aged_unaged_weight_proportions , expected_output )
+
+def test_compute_aged_sex_proportions( ):
```

Comment:
Making it identical to the function name
```suggestion
def test_compute_aged_sex_weight_proportions( ):
```